apps/admin_app/lib/admin_app_web/templates/option_type/form.html.eex

```diff
@@ -0,0 +1,5 @@
+<%= form_for @changeset, @action, fn f ->  %>
+  <%= input f, :name%>
+  <%= input f, :display_name%>
+  <%= submit "Submit", class: "btn btn-primary submit-btn" %>
```

Comment:
The position cannot be edited.

Reply:
Position is used to set the order of the option_type that will be displayed on UI.

For now we are not supporting changing the order as this input field will be hidden and will be update when the item is drag dropped in the UI through code(javascript).